webapp/src/components/MeetingAccordion.tsx

```diff
@@ -21,26 +22,42 @@ interface MeetingAccordionProps {
 const MeetingAccordion = ({ meeting }: MeetingAccordionProps) => {
   return (
     <Accordion>
-      <AccordionSummary expandIcon={<ExpandMoreIcon />}>
-        <ListItem>
-          <MuiLink
-            component={ReactRouterLink}
-            to={`/meetings/${meeting.id}`}
-            underline="none"
-            sx={{ color: 'text.primary' }}
-            onClick={e => {
-              e.stopPropagation();
+      <AccordionSummary
+        expandIcon={
+          <IconButton>
+            <ExpandMoreIcon />
+          </IconButton>
+        }
+        sx={{
+          '&:hover': {
+            backgroundColor: 'grey.200',
+          },
+        }}
+      >
+        <MuiLink
+          component={ReactRouterLink}
+          to={`/meetings/${meeting.id}`}
+          underline="none"
+          sx={{ color: 'text.primary' }}
+          onClick={e => {
+            e.stopPropagation();
+          }}
+        >
+          <Typography variant="body1">
+            Meeting on {formatDate(meeting.starts_at)}
+          </Typography>
+          <Typography
+            variant="body2"
+            sx={{
+              opacity: 0.87,
```

Comment:
Why is the opacity being set here? Shouldn't this be using a colour variant to change it to an appropriate shade of grey? I think `text.secondary` is the lighter text colour.

Reply:
Thanks for this.  I'll change the color.  

I only found the `Text legibility` documentation from MaterialDesign (see the Dark text on light backgrounds section) and followed the guidelines there.  (https://material.io/design/color/text-legibility.html#text-backgrounds).

Reply:
Yes, that's guidance for Material Design, which Material UI follows by defining a text palette that has reduced opacity black in two different shades which they call `text.primary` and `text.secondary`. The way to follow those guidelines when using Material UI is by using the appropriate colour from the palette—they've already done the work for us.

Reply:
OK, thank you for explaining.